js/windowsTitleBar/style.css

```diff
@@ -0,0 +1,99 @@
+#title-bar {
+    display: flex;
+    position: fixed;
+    background: #4a4a4a;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 32px;
+    padding-left: 10px;
+    justify-content: center;
+    align-items: center;
+    margin-left: -10px;
+    -webkit-app-region: drag;
+    -webkit-user-select: none;
+    box-sizing: content-box;
+    z-index: 1000;
+}
+
+#hamburger-menu-button {
+    color: white;
+    text-align: center;
+    width: 40px;
+    height: 32px;
+    background: none;
+    border: none;
+    border-image: initial;
+    display: inline-grid;
+    border-radius: 0;
+    padding: 11px;
+    box-sizing: border-box;
+    cursor: default;
```

Comment:
What does Slack do for this? Does it not use a pointer?

Reply:
Both on Slack and native Windows it is default cursor style 😄